backend/inspirehep/records/api/literature.py

```diff
@@ -288,56 +327,56 @@ def push_authors_phonetic_blocks_to_redis(self):
                     nx=True,
                 )
 
-    def add_files(self, data):
+    @staticmethod
+    def add_files(data):
         if not current_app.config.get("FEATURE_FLAG_ENABLE_FILES", False):
             LOGGER.info("Files are disabled")
             return data
 
         if "deleted" in data and data["deleted"]:
```

Comment:
If you already here, can you change this?
```suggestion
        if data.get("deleted", False):
```